### DISCUSSION TITLE
Uninstall

### DISCUSSION THREAD

**Discussion by MontrealPaul** [netalertx/NetAlertX#796] [Q&A]
How does one **UN**install NetAlertX, HW version?

I have an old Raspberry Pi V1, too weak to install Docker, so I tried installing the HW version. I tried various ways, i.e. wget, curl, etc., but I can't get anything to show up.
The app looks interesting on the website, but with all due respect, I must abandon this, and try one of the forks designed (and supported for) direct installation.

So, is there a command, or procedure, for a clean and complete removal?

Thank you!

P.S.: I searched discussions on various terms, such as uninstall, removal, delete, etc., and of course the documentation, but to avail, so if I missed it, my apologies.

**Comment by jokob-sk** [comment]
Uninstalling apps is one of the reasons I like docker, you know exactly where things are stored and it's easy to remove them :) 

Deleting the installation folder and nginx directory should remove most of the installation artifacts. These should be:

- /var/www/html/netalertx
- /app

Leiweibau is great at supporting the HW install, so you will be in good hands!

Thanks for giving it a go!

**Comment by MontrealPaul** [comment]
Wow, jokob, thank you for this super fast reply!
No need to sell me on Docker, but my poor little RBP1 is too weak to run it. One day, if/when I upgrade, it'll be fun to check out NetAlertX!

Cheers,
 -Paul

**Reply by jokob-sk** [reply]
Great, looking forward! :)
